Gems/ScriptCanvas/Code/Editor/View/Windows/mainwindow.ui

```diff
@@ -167,7 +167,6 @@
    <addaction name="menuView_2"/>
    <addaction name="menuTools_2"/>
    <addaction name="menuSettings_2"/>
-   <addaction name="menuScript_Events_PREVIEW"/>
```

Comment:
thanks! I missed this one

Reply:
I can't take credit, Qt Designer found it for me :)